misc/dist/ios_xcode/godot_ios/godot_ios-Info.plist

```diff
@@ -26,6 +26,8 @@
 	<string>$signature</string>
 	<key>CFBundleVersion</key>
 	<string>$version</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
```

Comment:
For the reference: https://developer.apple.com/documentation/security/complying_with_encryption_export_regulations

Reply:
Godot includes some encryption features, both in its thirdparty libraries and some of it are exposed to the public scripting API (notably via the `Crypto` singleton). Did you assess if they fall within the "exempt" category for Apple?

Reply:
As far as I've seen from the source code, Godot does not use any self-written encryption algorithm, so it's should be ok. So far any app that I've published was okay with using any standard encryption.